modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -405,59 +408,213 @@ protected function register_controls() {
 			'tab' => Controls_Manager::TAB_STYLE,
 		] );
 
-		$this->add_control( 'border_width', [
-			'label' => esc_html__( 'Border Width', 'elementor' ),
+		$this->add_responsive_control( 'tabs_title_spacing', [
+			'label' => esc_html__( 'Spacing', 'elementor' ),
 			'type' => Controls_Manager::SLIDER,
-			'default' => [
-				'size' => 1,
-			],
 			'range' => [
 				'px' => [
 					'min' => 0,
-					'max' => 10,
+					'max' => 400,
 				],
 			],
-			'selectors' => [
-				'{{WRAPPER}}' => '--tabs-v2-border-width: {{SIZE}}{{UNIT}};',
+			'default' => [
+				'unit' => 'px',
```

Comment:
If the only size units you set are PX, why do you need this?

Reply:
updated